models/Event.js

```diff
@@ -12,14 +12,15 @@ module.exports =
 
 const EventSchema = new Schema(
   {
-    name: String,
-    date: Date,
-    attendance: Number,
-    description: String,
-    images: [String],
-    category: [String],
-    organizerId: String,
-    eventStalls: [eventStallSchema],
+    name: { type: String, required: true },
+    date: { type: String, required: true },
+    time: { type: String, required: true },
+    expectedAttendance: { type: Number, required: true },
+    description: { type: String, required: true },
+    images: { type: [String], default: undefined, required: true },
+    eventCategory: { type: [String], default: undefined, required: true },
+    organizerId: { type: String, required: true },
```

Comment:
Why no ref?

Reply:
The other PR hasn't been merged. I left a comment on it. Check it out

Reply:
It should be the ref